### ISSUE TITLE
Question about the structure of the dataset

### ISSUE THREAD

**Issue by lanxx314** [hexiao0275/LCMA_RGBT#1]
Thanks for your contributions, can you share the training dataset after reorganizing the directory structure?

**Comment by hexiao0275** [comment]
see https://github.com/hexiao0275/CALNet-Dronevehicle , , at Readme.md contains download links to our carefully processed DroneVehicle dataset.

Processed labels for our carefully organized dataset:

-[Label]  [download](https://pan.baidu.com/s/17tLn0D6yZkVqokMBpis1jw) password:5sow

Pre-training weights download:

-[Weight]  [download](https://pan.baidu.com/s/1PnmdKqIxPnTgK6yQ6WfwpA) password:zvi2
